### PR TITLE
ChineseLocalization

### PR DESCRIPTION
**Pull request by xianli123**
As a Chinese native speaker, I reviewed all Chinese localization files and optimized the localization wordings. This branch includes the original localization and some updates.

### CODE REVIEW

**Comment by jonkoops** [comment]
@xianli123 do we have another native mandarin speaker that could review this change?

**Comment by xianli123** [comment]
Hi @jonkoops I can ask our UXD team members if anyone has time to help check it. Once all things are done, I will let you know.

File: js/apps/admin-ui/public/resources/zh-CN/authentication-help.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "flowType": "它是什么样的形式",
-  "topLevelFlowType": "什么样的顶层流是什么？类型“客户端”用于客户端（应用程序）的认证，当通用是为用户和其他",
+  "topLevelFlowType": "流程有什么样的类型？“客户端”类型用于客户端（应用程序）的认证，而通用的类型是用于用户和其他的认证",
```
Comment (xianli123) [inline js/apps/admin-ui/public/resources/zh-CN/authentication-help.json:3]:
```suggestion
  "topLevelFlowType": "顶层流程有什么样的类型？“客户端”类型用于客户端（应用程序）的认证，而通用的类型是用于用户和其他的认证",
```

**Review by edewit — DISMISSED** [review]

**Comment by xianli123** [comment]
@edewit Thanks for approving it. I have invited Haley to help review the translations and wording. Once all things are done, I will let you know.

**Comment by jonkoops** [comment]
@xianli123 any progress on this PR?

**Comment by xianli123** [comment]
Hi @jonkoops I invited Haley to help with the review. Today, we had a meeting and talked about this pr. She will finish reviewing it as soon as possible (it could be by the end of next week). Once all things are done, I will let you know. Thanks.

File: js/apps/admin-ui/public/resources/zh-CN/authentication-help.json [hunk 2]
```diff
@@ -1,9 +1,51 @@
...
\ No newline at end of file
+  "name": "新流程命名的帮助文本",
+  "description": "新流程描述的帮助文本",
+  "createFlow": "您可以在此表单中创建顶级流程",
+  "flowType": "它是一种什么样的形式？",
+  "topLevelFlowType": "它是什么类型的顶级流程？客户端类型用于客户端（应用程序）的身份验证，而通用型流程用于用户和其他所有内容的验证。",
+  "addExecution": "执行可以有多种操作，从发送重置电子邮件到验证 OTP。",
+  "addSubFlow": "子级流程可以是一般形式的，也可以是表单形式的。表单类型用于构造子流，为用户生成单个流程。子级流程是一种特殊类型的执行，根据它们包含的执行的评估方式评估流程成功与否。",
+  "alias": "配置名称",
+  "authDefaultActionTooltip": "如果启用，任何新用户都将获得分配给它的此项必需操作。",
```
Comment (xianli123) [inline js/apps/admin-ui/public/resources/zh-CN/authentication-help.json:10]:
```suggestion
  "authDefaultActionTooltip": "如果启用，此项操作对于任何新用户都将是必需的操作。",
```

**Comment by jonkoops** [comment]
@xianli123 could you do a rebase of this branch on `main`? It seems some conflicts have started to appear.

**Comment by jonkoops** [comment]
@xianli123 what I mean is that new changes have landed on the `main` branch, that are in conflict with the changes you have here. So in that case you will have to get the latest from `main` and resolve the conflicts. Probably the easiest way to do this is to use the [GitHub desktop app](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/keeping-your-local-repository-in-sync-with-github/syncing-your-branch-in-github-desktop#rebasing-your-project-branch-onto-another-branch).

Feel free to let me know if you need some help, we can hop on a call together.

**Comment by xianli123** [comment]
@jonkoops I see. I will try to solve the conflict. If there is any problem, I will ask for your help. Thanks.🙏

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 3]
```diff
@@ -1,113 +1,900 @@
...
+  "updateMessageBundleSuccess": "成功！消息包已更新。",
+  "updateMessageBundleError": "更新消息包时出错。",
+  "addMessageBundleError": "创建消息包时出错，{{error}}",
+  "allGroups": "所有组",
+  "attributeName": "姓名",
+  "attributeDisplayName": "显示名称",
+  "attributeGroup": "属性组",
+  "enabledWhen": "启用时间",
+  "required": "必需",
+  "requiredFor": "被应用于",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:359]:
```suggestion
  "requiredFor": "针对何人为必填项",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 4]
```diff
@@ -1,113 +1,900 @@
...
+    "colDescription": "描述"
   },
   "validatorColNames": {
-    "colConfig": "Config"
+    "colName": "验证者姓名",
+    "colConfig": "设置"
   },
+  "deleteValidatorConfirmTitle": "删除验证器？",
+  "deleteValidatorConfirmMsg": "您确定要永久删除验证器 {{validatorName}} 吗？",
+  "validatorDeletedSuccess": "用户用户资料配置已保存。",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:383]:
```suggestion
  "validatorDeletedSuccess": "用户资料配置已保存。",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 5]
```diff
@@ -1,113 +1,900 @@
...
+  "createAttributeSubTitle": "创建新属性",
+  "createAttributeSuccess": "成功！用户资料的配置已保存。",
+  "createAttributeError": "错误！用户资料的配置尚未保存{{error}}。",
+  "attributesDropdown": "属性下拉框",
+  "deleteAttributeConfirmTitle": "删除属性？",
+  "deleteAttributeConfirm": "您确定要永久删除属性 {{attributeName}} 吗？",
+  "deleteAttributeSuccess": "属性已删除",
+  "deleteAttributeError": "属性未删除",
+  "always": "总是",
+  "scopesAsRequested": "请求范围",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:399]:
```suggestion
  "scopesAsRequested": "在此范围内生效",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 6]
```diff
@@ -1,113 +1,900 @@
...
+  "generalSettings": "通用设置",
+  "permission": "权限",
+  "validations": "验证",
+  "annotations": "注释",
+  "addAnnotationText": "添加注释",
+  "validateName": "不允许未命名的属性配置。",
+  "eventType": "被保存事件的类型",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:406]:
```suggestion
  "eventType": "被保存的事件类型",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 7]
```diff
@@ -1,113 +1,900 @@
...
+    "CLIENT_LOGIN": {
+      "name": "客户端登录",
+      "description": "客户端登录"
+    },
+    "RESET_PASSWORD_ERROR": {
+      "name": "重置密码错误",
+      "description": "重置密码错误"
+    },
+    "IMPERSONATE_ERROR": {
+      "name": "模拟错误",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:448]:
```suggestion
      "name": "角色扮演错误",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 8]
```diff
@@ -1,113 +1,900 @@
...
+      "name": "自定义所需操作",
+      "description": "自定义所需操作"
+    },
+    "RESTART_AUTHENTICATION": {
+      "name": "重启认证",
+      "description": "重启认证"
+    },
     "IMPERSONATE": {
-      "name": "模拟"
+      "name": "模拟",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:464]:
```suggestion
      "name": "角色扮演",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 9]
```diff
@@ -1,113 +1,900 @@
...
+      "description": "模拟"
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:465]:
```suggestion
      "description": "角色扮演"
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 10]
```diff
@@ -1,113 +1,900 @@
...
+    "LOGIN": {
+      "name": "登录",
+      "description": "登录"
+    },
+    "UPDATE_PASSWORD_ERROR": {
+      "name": "更新密码错误",
+      "description": "更新密码错误"
+    },
+    "CLIENT_INITIATED_ACCOUNT_LINKING": {
+      "name": "客户发起的账户关联",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:480]:
```suggestion
      "name": "客户端发起的账号关联",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 11]
```diff
@@ -1,113 +1,900 @@
...
+      "description": "客户发起的账户链接"
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:481]:
```suggestion
      "description": "客户端发起的账号关联"
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 12]
```diff
@@ -1,113 +1,900 @@
...
+    "DELETE_ACCOUNT_ERROR": {
+      "name": "删除账号错误",
+      "description": "删除账号错误"
+    },
+    "CLIENT_REGISTER": {
+      "name": "客户端注册",
+      "description": "客户端注册"
+    },
+    "IDENTITY_PROVIDER_LINK_ACCOUNT": {
+      "name": "身份提供者链接账户",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:504]:
```suggestion
      "name": "身份提供商关联账号",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 13]
```diff
@@ -1,113 +1,900 @@
...
+      "description": "身份提供者链接账户"
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:505]:
```suggestion
      "description": "身份提供商关联账号"
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 14]
```diff
@@ -1,113 +1,900 @@
...
+      "name": "删除联合身份",
+      "description": "删除联合身份"
+    },
+    "IDENTITY_PROVIDER_POST_LOGIN": {
+      "name": "身份供应者非首次登录",
+      "description": "身份供应者非首次登录"
+    },
+    "IDENTITY_PROVIDER_LINK_ACCOUNT_ERROR": {
+      "name": "身份供应者链接账号错误",
+      "description": "身份供应者链接账号错误"
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:585]:
```suggestion
      "description": "身份提供商关联账号错误"
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 15]
```diff
@@ -1,113 +1,900 @@
...
+      "name": "撤销授权错误",
+      "description": "撤销授权错误"
+    },
+    "EXECUTE_ACTION_TOKEN": {
+      "name": "执行动作令牌",
+      "description": "执行动作令牌"
+    },
+    "LOGOUT_ERROR": {
+      "name": "注销错误",
+      "description": "注销错误"
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:605]:
```suggestion
      "description": "登出错误"
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 16]
```diff
@@ -1,113 +1,900 @@
...
+    "VERIFY_PROFILE": {
+      "name": "验证个人资料",
+      "description": "验证个人资料"
+    },
+    "VERIFY_PROFILE_ERROR": {
+      "name": "验证配置文件错误",
+      "description": "验证配置文件错误"
+    },
+    "OAUTH2_DEVICE_CODE_TO_TOKEN": {
+      "name": "Oauth2设备代码令牌",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:788]:
```suggestion
      "name": "Oauth2设备用于交换令牌的代码",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 17]
```diff
@@ -1,113 +1,900 @@
...
+      "description": "Oauth2设备代码令牌"
+    },
+    "OAUTH2_DEVICE_CODE_TO_TOKEN_ERROR": {
+      "name": "Oauth2设备代码令牌错误",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:792]:
```suggestion
      "name": "Oauth2设备用于交换令牌的代码错误",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 18]
```diff
@@ -1,113 +1,900 @@
...
+      "description": "Oauth2设备代码令牌错误"
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:793]:
```suggestion
      "description": "Oauth2设备用于交换令牌的代码错误"
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 19]
```diff
@@ -1,113 +1,900 @@
...
+    },
+    "OAUTH2_DEVICE_VERIFY_USER_CODE": {
+      "name": "Oauth2设备验证用户密码",
+      "description": "Oauth2 设备验证用户代码"
+    },
+    "OAUTH2_DEVICE_VERIFY_USER_CODE_ERROR": {
+      "name": "Oauth2设备验证用户密码错误",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:800]:
```suggestion
      "name": "Oauth2设备验证用户代码错误",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 20]
```diff
@@ -1,113 +1,900 @@
...
+  "FAIL": "导入失败",
+  "SKIP": "跳过",
+  "OVERWRITE": "覆盖",
+  "added": "添加",
+  "skipped": "被跳过",
+  "overwritten": "被覆盖",
+  "importAdded_zero": "没有添加记录。",
+  "importAdded_one": "添加了一条记录。",
+  "importAdded_other": "已添加 {{count}} 条记录。",
+  "importOverwritten_zero": "没有记录被覆盖。",
```
Comment (yih-wang) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:894]:
```suggestion
  "importOverwritten_zero": "没有记录被覆写。",
```

**Review by yih-wang — CHANGES_REQUESTED** [review]

File: js/apps/admin-ui/public/resources/zh-CN/client-scopes-help.json [hunk 21]
```diff
@@ -1,10 +1,18 @@
...
-  "tokenClaimName": "token中加入的申请者名. 这可以是个完整的分级信息例如 'address.street'. 这种情况下，会生成一个复杂的json回复",
-  "claimJsonType": "用来展现申请的JSON 类型 long, int, boolean, 和 String 是有效值",
-  "protocolMapper": "协议..."
-}
\ No newline at end of file
+  "name": "客户端作用域的名称。在领域中必须是唯一的。名称不应包含空格字符，因为它用作作用域参数的值。",
+  "description": "客户作用域说明",
+  "protocol": "此客户端作用域提供的SSO协议配置",
+  "type": "客户端作用域，将以默认作用域的形式添加到每个创建的客户端",
+  "displayOnConsentScreen": "如果启用，并且此客户端作用域被添加到需要许可的某些客户端，则“许可页面的显示文本”指定的文本将显示在许可申请的页面上。如果禁用，此客户端作用域将不会显示在许可申请的页面上",
```
Comment (xianli123) [inline js/apps/admin-ui/public/resources/zh-CN/client-scopes-help.json:6]:
```suggestion
  "displayOnConsentScreen": "如果启用，并且此客户端作用域被添加到需要许可的某些客户端，则“许可页面的展示文本”指定的文本将展示在许可申请的页面上。如果禁用，此客户端作用域将不会展示在许可申请的页面上",
```

File: js/apps/admin-ui/public/resources/zh-CN/realm-settings.json [hunk 22]
```diff
@@ -1,113 +1,900 @@
...
+  "removeConfirm_one": "你确定要移除这个群组",
+  "removeConfirm_other": "你确定要删除这些组吗。",
+  "groupRemove_one": "群组已删除",
+  "groupRemove_other": "群组已移除",
+  "groupRemoveError": "删除群组 {error} 时出错",
+  "defaultGroupAdded_one": "新组添加到默认群组",
+  "defaultGroupAdded_other": "将 {{count}} 个群组添加到默认群组。",
+  "defaultGroupAddedError": "将群组添加到默认群组时出错 {error}。",
+  "noDefaultGroups": "没有默认群组",
+  "noDefaultGroupsInstructions": "默认组允许您在整个<1>身份代理</1>过程中创建或导入任何新用户时自动分配组成员。添加默认组以开始使用。",
```
Comment (xianli123) [inline js/apps/admin-ui/public/resources/zh-CN/realm-settings.json:852]:
```suggestion
  "noDefaultGroupsInstructions": "设置默认组将允许您在<1>identity brokering</1>过程中创建或导入新用户时自动分配群组。添加默认组以开始使用这一功能。",
```

**Comment by xianli123** [comment]
Hi @jonkoops @ssilvert I closed this pr due to some conflicts and pull a new PR https://github.com/keycloak/keycloak/pull/20740. The main files have been reviewed by Haley and I updated them according to her suggestions. Please take a look and merge it if there is no error. Thanks!

**Pull request closed without merging**